### PR TITLE
Updated example on data containers.

### PR DESCRIPTION
**Pull request by jaeilepp**
Integration from https://github.com/mne-tools/mne-python/issues/3725.

### CODE REVIEW

File: examples/io/plot_objects_from_arrays.py [hunk 1]
```diff
@@ -95,6 +99,35 @@
...
+epochs.plot(scalings='auto', block=True)
+
+###############################################################################
+# Create overlapping epochs using :func:`mne.make_fixed_length_events` (50 %
+# overlap). This also roughly doubles the amount of events compared to the
+# previous event list.
+
+duration = 0.5
+events = mne.make_fixed_length_events(raw, event_id, duration=duration)
+print events
```
Comment (agramfort) [inline examples/io/plot_objects_from_arrays.py]: print(events)

File: examples/io/plot_objects_from_arrays.py [hunk 2]
```diff
@@ -95,6 +99,35 @@
...
+
+###############################################################################
+# Create overlapping epochs using :func:`mne.make_fixed_length_events` (50 %
+# overlap). This also roughly doubles the amount of events compared to the
+# previous event list.
+
+duration = 0.5
+events = mne.make_fixed_length_events(raw, event_id, duration=duration)
+print(events)
+epochs = mne.Epochs(raw, events=events, tmin=tmin, tmax=tmax, verbose=True)
```
Comment (mmagnuski) [inline examples/io/plot_objects_from_arrays.py]: same here: `baseline=None` if tmin is zero.

File: examples/io/plot_objects_from_arrays.py [hunk 3]
```diff
@@ -60,6 +63,7 @@
 ###############################################################################
 # EpochsArray
 
+# This is used to identify the events
```
Comment (mmagnuski) [inline examples/io/plot_objects_from_arrays.py]: this is not super-clear, maybe better to say that events array is created where the first row defines event time in samples and the last column specifies event identity?

**Comment by mmagnuski** [comment]
👍

**Comment by agramfort** [comment]
thx heaps @jaeilepp

**Pull request merged**